.github/workflows/interop-test.yml

```diff
@@ -1,9 +1,22 @@
 name: Interoperability Testing
 on:
+  workflow_dispatch:
```

Comment:
What is this doing?

Reply:
it enables a workflow to be triggered [manually](https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow#configuring-a-workflow-to-run-manually).